src/WebJobs.Script/Diagnostics/HealthChecks/HealthCheckTags.cs

```diff
@@ -8,30 +8,44 @@ namespace Microsoft.Azure.WebJobs.Script.Diagnostics.HealthChecks
     /// </summary>
     internal static class HealthCheckTags
```

Comment:
We can introduce a hierarchical structure to health check tags like HealthCheckTags.Area.Connectivity and HealthCheckTags.ConfigurationSection.Storage